scheduler/plugin/mask.go

```diff
@@ -233,6 +235,7 @@ func maskInterfaces(plugin *wasmPlugin) (framework.Plugin, error) {
 type basePlugin interface {
 	framework.EnqueueExtensions
 	framework.PreFilterPlugin // to implement cycle state reset
+	framework.PreFilterExtensions
```

Comment:
@sanposhiho 
This is not necessary?

Reply:
Yeah, PreFilterExtensions isn't something wasm must have.